service/src/main/kotlin/fi/espoo/evaka/messaging/MessageQueries.kt

```diff
@@ -1053,7 +1091,37 @@ fun Database.Read.getReceiversForNewMessage(
                 MessageReceiversResponse(accountId = accountId, receivers = accountReceivers)
             }
 
-    return municipalReceivers + unitReceivers
+    val serviceWorkerReceivers =
+        createQuery(
+                """
+        WITH accounts AS (
+            SELECT id, type, daycare_group_id, employee_id, person_id FROM message_account
+            WHERE id = ANY(:accountIds) AND type = 'SERVICE_WORKER'::message_account_type
+        )
+        SELECT acc.id AS account_id, p.id as person_id, p.first_name, p.last_name 
+        FROM accounts acc, person p
+        JOIN application app ON p.id = app.guardian_id
+        WHERE app.status = 'SENT'
```

Comment:
This query is probably going to be slow. There's no suitable index on `application.status`, so it will probably be a seq scan on either `person` or `application`.

Is this a temporary query or will it be there also when the feature is ready?